cmd/zed/lake/ztests/log.yaml

```
@@ -0,0 +1,33 @@
+script: |
+  export ZED_LAKE_ROOT=test
+  zed lake init -q
+  zed lake create -q -p POOL
+  zed lake load -date 2021-05-28T15:13:42Z -user testuser -q -p POOL 1.zson
+  zed lake load -date 2021-05-28T15:13:43Z -user testuser -q -p POOL 2.zson
```

Nit: This is a little easier to read if `-q -p POOL` always comes first.
```suggestion
  zed lake load -q -p POOL -date 2021-05-28T15:13:42Z -user testuser 1.zson
  zed lake load -q -p POOL -date 2021-05-28T15:13:43Z -user testuser 2.zson
```